Modules/Sources/ShareExtensionCore/ShareExtensionConfiguration.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import BuildSettingsKit
+
+extension BuildSettings {
+    public var shareExtensionConfiguration: ShareExtensionConfiguration {
+        switch brand {
+        case .wordpress: return .wordpress
+        case .jetpack: return .jetpack
+        }
+    }
+}
+
+public struct ShareExtensionConfiguration: Sendable {
+    public var keychainUsernameKey: String
+    public var keychainTokenKey: String
+    public var keychainServiceName: String
+    public var userDefaultsPrimarySiteName: String
+    public var userDefaultsPrimarySiteID: String
+    public var userDefaultsLastUsedSiteName: String
+    public var userDefaultsLastUsedSiteID: String
+    public var maximumMediaDimensionKey: String
+    public var recentSitesKey: String
+
+    static let wordpress = ShareExtensionConfiguration(
+        keychainUsernameKey: "Username",
+        keychainTokenKey: "OAuth2Token",
+        keychainServiceName: "ShareExtension",
```

Comment:
I wanted to add a "WPShareExtensionPrefix" parameter to `Info.plist` but because some keys have prefixes and some don't, I decided to just move the existing code without changing it.